backend/models/user.model.test.js

```diff
@@ -0,0 +1,165 @@
+// import necessary modules
+const mongoose = require('mongoose');
+const { User } = require('./user.model');
+// MongoDB Memory Server is used for testing purposes to avoid using a real database
+const { MongoMemoryServer } = require('mongodb-memory-server');
+
+
+describe('Unit tests for User Model', () => {
+  let mongoServer;
```

Comment:
We should remove all of this setup for the mongoServer because this was the source of intermittent errors previously.